risc0/binfmt/src/image.rs

```diff
@@ -345,17 +367,16 @@ impl MemoryImage {
 
 impl Default for Page {
     fn default() -> Self {
-        Self::from_vec(vec![0; PAGE_BYTES])
+        Self::from_arr([0; PAGE_BYTES])
```

Comment:
I think this is actually not great, IIRC this will end up allocating 1k of zeros on the stack and then copy it to the heap, there are ways around this I think maybe using some `unsafe`, but then again if its not showing up on your trace maybe it doesn't matter 🤷